core/src/main/java/com/netflix/conductor/core/metadata/MetadataMapperService.java

```diff
@@ -78,13 +80,13 @@ public WorkflowDef lookupForWorkflowDefinition(String name, Integer version) {
 
     @VisibleForTesting
     Optional<WorkflowDef> lookupWorkflowDefinition(String workflowName, int workflowVersion) {
-        ServiceUtils.checkNotNullOrEmpty(workflowName, "Workflow name must be specified when searching for a definition");
+        Preconditions.checkArgument(StringUtils.isNotBlank(workflowName), "Workflow name must be specified when searching for a definition");
```

Comment:
Why changing it back to Preconditions?

Reply:
This is an internal method so reverted back to use Preconditions instead of using Service Utils which throws ApplicationException with http code.